umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops-pwsh.md

```diff
@@ -0,0 +1,147 @@
+---
+description: >-
+  This section provides a step-by-step guide to setting up a CI/CD pipeline in
+  Azure DevOps using the provided sample Powershell scripts.
+---
+
+# Azure DevOps with Powershell scripts
+
+Before setting up the pipeline in Azure Devops, make sure that the steps in the [Configuring a CI/CD pipeline](./) are done.
+
+You will need to define your pipeline in YAML, and find a way to interact with the Umbraco Cloud Api.
+
+
+{% hint style="info" %}
```

Comment:
this part isn't rendered correctly in the markdown preview

Reply:
is it the {% hint ....%} part?I guess this is GitBook magic :) It should look correct once merged into docs